hack/update-codegen.sh

```diff
@@ -108,7 +100,7 @@ ${REPO_ROOT_DIR}/hack/update-deps.sh
 ${REPO_ROOT_DIR}/hack/update-openapigen.sh
 
 # Make sure the generated API reference docs are up-to-date
-${REPO_ROOT_DIR}/hack/update-reference-docs.sh
+# ${REPO_ROOT_DIR}/hack/update-reference-docs.sh
```

Comment:
I need to understand why this fails, but I am a bit worried on this as we depend on a fork that is most likely not maintained…

cc @waveywaves @afrittoli @tektoncd/core-maintainers